16_results_and_eval/foraging.tex

```diff
@@ -0,0 +1,25 @@
+\section{Foraging Simulation}
+\subsection{Foraging Simulation Results}
+\begin{figure}[!htb]
+    \centering
+    \includegraphics[width=0.95\textwidth]{16_results_and_eval/images/ForageCommonSim.png}
+    \caption{Simulation results for Foraging Parameters using the Environmental metrics of the Archipelago.}
+    \label{fig:Resource}
+\end{figure}
+\subsection{Discussion on Simulation Results}
+The simulation for foraging has been completed by adjusting a single environmental parameter whilst fixing the rest and comparing how the metrics change relative to the baseline, examined in Section~\ref{sec:Simulations:baseline}. For example, in Figure \ref{fig:Resource} the first test involved decreasing the parameter of \texttt{foragingDeerMaxPerHunt} and recording the change observed on the metrics. The tests were run thrice so that extreme scenarios could be mitigated to some extent. If two consecutive tests returned similar results, the result was then recorded. Our findings are presented in Figure \ref{fig:Resource}. The list of metrics recorded was larger, but some were discarded because no strong correlation was visible and others were just irrelevant. 
```

Comment:
```suggestion
The simulation for foraging has been completed by adjusting a single environmental parameter whilst fixing the rest and comparing how the metrics change relative to the baseline, examined in Section~\ref{sec:Simulations:baseline}. For example, in Figure \ref{fig:Resource} the first test involved decreasing the parameter of \texttt{foragingDeerMaxPerHunt} and recording the change observed on the metrics. The tests were run thrice so that extreme scenarios could be mitigated to some extent. If two consecutive tests returned similar results, the result was then recorded. Our findings are presented in Figure~\ref{fig:Resource}. The list of metrics recorded was larger, but some were discarded because no strong correlation was visible and others were just irrelevant. 
```